mod-source-record-manager-server/src/main/java/org/folio/dao/JournalRecordDao.java

```diff
@@ -60,12 +60,13 @@ public interface JournalRecordDao {
    * @param jobExecutionId job execution id
    * @param sortBy         sorting criteria
    * @param order          sorting direction
+   * @param errorsOnly     filtering by error field
    * @param limit          limit
    * @param offset         offset
    * @param tenantId       tenantId
    * @return future with JobLogEntryDto collection
    */
-  Future<JobLogEntryDtoCollection> getJobLogEntryDtoCollection(String jobExecutionId, String sortBy, String order, int limit, int offset, String tenantId);
+  Future<JobLogEntryDtoCollection> getJobLogEntryDtoCollection(String jobExecutionId, String sortBy, String order, boolean errorsOnly, int limit, int offset, String tenantId);
```

Comment:
Maybe to move errorsOnly param to the end of param's list to be it consistent with order of params in sql function ?